### DISCUSSION TITLE
Wrong Stackoverflow links

### DISCUSSION THREAD

**Discussion by pilipeikoko** [community/community#14212] [Copilot Conversations] [answered]
When Copilot suggests stackoverflow links, it might be invalid and you are being redirected to wrong thread. 

**Real example in c# :**
**My comment** : android permission to storage

**Suggest links** :  
```
https://stackoverflow.com/questions/26491644/android-permission-to-access-external-storage
https://stackoverflow.com/questions/52724/how-do-i-make-my-android-application-use-the-sd-card
```

**Result** : If you check for any of these links, you are redirected to the wrong thread

**Comment by MalvinHiepp** [comment, marked as answer]
I think the problem isn't at Github copilot, but with Stackoverflow itself. On myself, I have the same problems on different threads. I guess it depends on the thread. I am not sure though. In the end, I get these problems even without a Github copilot.

**Reply by DavraYoung** [reply]
Yes, Copilot certainly generates fake urls, for example:
```
Question from Human
How much time will it take to train small OpenAI Whisper on RTX 3080TI?

Answer from Copilot:
It will take 1.5 days to train small OpenAI Whisper on RTX 3080TI.
Details: https://arxiv.org/abs/2106.08250```
